operator/src/main/java/org/bf2/operator/controllers/ManagedKafkaAgentController.java

```diff
@@ -44,9 +42,7 @@
  */
 @ApplicationScoped
 @ControllerConfiguration(
-        finalizerName = Constants.NO_FINALIZER,
```

Comment:
@MikeEdgar The updated operator sdk doesn't have the same constants. Are we using this `finalizerName` value somewhere else?

Reply:
Remove it - it's not needed in the newer version. The docs say it will only add a finalizer if the `Cleaner` interface is implemented by the controller.

Reply:
Sure. It's already removed!